heka/sandbox/decoders/extract_telemetry_dimensions.lua

```diff
@@ -55,7 +55,7 @@ function process_message()
     -- them later with values from the parsed JSON Payload if available.
     -- These fields should match the ones specified in the namespace_config for
     -- the "telemetry" endpoint of the HTTP Edge Server.
-    msg.Fields.docId            = read_message("Fields[DocumentID]")
+    msg.Fields.documentId       = read_message("Fields[documentId]")
     msg.Fields.docType          = read_message("Fields[reason]")
```

Comment:
We can make the other 2 mismatched field names consistent by updating the `namespace_config` to use `docType` and `appBuildId` instead of `reason` and `appBuildID` respectively.  The HTTP Edge names are set to match old telemetry naming, but we might as well update it to match v4 naming.